README.md

```diff
@@ -65,14 +74,17 @@ To determine whether to allow response _response_ to a request _request_, run th
 1. If _mimeType_ is not failure, then:
    1. If _mimeType_ is an opaque-safelisted MIME type, then return true.
    1. If _mimeType_ is an opaque-blocklisted-never-sniffed MIME type, then return false.
-   1. If _response_'s status is `206` and _mimeType_ is an opaque-blocklisted MIME type, then return false. TODO: is this needed with the requesters set?
+   1. If _response_'s status is 206 and _mimeType_ is an opaque-blocklisted MIME type, then return false.
```

Comment:
> This step stems from https://fetch.spec.whatwg.org/#corb-check.

Doh... I forgot about it.  Yes, Chrome's CORB implementation still blocks 206 responses with html/json/xml MIME types.  Given this, I think we should just land the PR in its current shape.

> Are you suggesting [...]

I was just trying to argue why I think the earlier step (blocking 206 for html/xml/json) might be redundant, because latter steps (without any changes) would block such html/xml/json responses.